dotnet6/cookiecutter-aws-sam-quick-start-from-scratch-dotnet/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -0,0 +1,33 @@
+# This is the SAM template that represents the architecture of your serverless application
+# https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-template-basics.html
+
+# The AWSTemplateFormatVersion identifies the capabilities of the template
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/format-version-structure.html
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  {{cookiecutter.project_name}}
+
+# Transform section specifies one or more macros that AWS CloudFormation uses to process your template
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/transform-section-structure.html
+Transform:
+- AWS::Serverless-2016-10-31
+
+# Resources declares the AWS resources that you want to include in the stack
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html
+Resources:
+  # Each Lambda function is defined by properties:
+  # https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+
+  # This is a Lambda function config associated with the source code: hello-from-lambda.js
+  helloFromLambdaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
```

Comment:
@rjesh-git since you've added the architecture from cookiecutter.json file, please add it to the template,
here's an example 
[https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/%7B%7Bcookiecutter.project_name%7D%7D/template.yaml#L68](https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/%7B%7Bcookiecutter.project_name%7D%7D/template.yaml#L68)

Reply:
✅